### PR TITLE
PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by lucasmichot**
- Add a basic .editorconfig file
- Use the cache feature so that we only try to fix files that I've changed since the last fix
- Ensure this .php_cs.cache files is not commited

### CODE REVIEW

File: src/Illuminate/Console/Scheduling/CallbackEvent.php [hunk 1]
```diff
@@ -58,6 +58,8 @@ public function run(Container $container)
 
         try {
             $response = $container->call($this->callback, $this->parameters);
+        } catch (Exception $e) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Console/Scheduling/CallbackEvent.php]: no

Comment (GrahamCampbell) [inline src/Illuminate/Console/Scheduling/CallbackEvent.php]: this was removed on purpose

File: .php_cs [hunk 2]
```diff
@@ -3,7 +3,6 @@
 $finder = Symfony\Component\Finder\Finder::create()
     ->files()
     ->in(__DIR__)
-    ->name('*.stub')
```
Comment (GrahamCampbell) [inline .php_cs]: no, keep this

File: src/Illuminate/Console/Scheduling/CallbackEvent.php (hunk 1 above)
Comment (lucasmichot) [inline src/Illuminate/Console/Scheduling/CallbackEvent.php]: Looks strange to provide a throw docblock and hiding the throwing of this exception

Comment (GrahamCampbell) [inline src/Illuminate/Console/Scheduling/CallbackEvent.php]: Taylor already decided he wants it like that. Please revert that change.

Comment (lucasmichot) [inline src/Illuminate/Console/Scheduling/CallbackEvent.php]: sure

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

**Pull request merged**